README.md

```diff
@@ -45,9 +45,12 @@ We will implement a new parallel blobs submission mechanism which is compatible
 
 ### 2. Failover (for Liveness)
 
-The upstream altda code does not support failover. If the EigenDA network goes down, the rollup will be stuck.
+We have implemented an automatic failover mechanism that ensures rollup liveness if EigenDA is unavailable:
 
-We will implement a failover mechanism to allow the rollup to continue processing transactions even if the EigenDA network is down.
+- **Automatic Detection**: The batcher detects EigenDA unavailability when the proxy returns HTTP 503 (Service Unavailable) responses
+- **Seamless Failover**: When EigenDA is down, the batcher automatically switches to posting batch data as Ethereum calldata instead
+- **Automatic Failback**: Once EigenDA becomes available again, the batcher automatically returns to using EigenDA for cost efficiency
```

Comment:
actually afair failover happens for a single channel. So there is no failback. Every channel is first sent to proxy, if 503 is returned, it fails over.